recurring_contract/models/recurring_contract.py

```diff
@@ -233,8 +231,14 @@ def create(self, vals):
         if vals.get('reference', '/') == '/':
             vals['reference'] = self.env['ir.sequence'].next_by_code(
                 'recurring.contract.ref')
-
-        return super().create(vals)
+        res = super().create(vals)
+        # Define if the invoices should temporarily be suspended
+        today = fields.Date.today()
+        day_block_inv = self.env["ir.config_parameter"].sudo().get_param(
+            f"compassion_nordic_accounting.inv_block_day_{self.env.company.id}", 31)
+        if today.day > day_block_inv:
+            self.mapped("group_id").write({"invoice_suspended_until": today + relativedelta(months=1)})
+        return res
```

Comment:
I would rather implement this logic in a `@api.onchange` method or as default value method, so that the date can be visible in the user interface before saving the document. Otherwise the date will be set after we save the contract and the user may not notice (He may still want to be in control of that).